src/styles/main.scss

```diff
@@ -0,0 +1,10 @@
+@import 'variables';
+@import 'calendar';
```

Comment:
Ensure that the 'variables' and 'calendar' SCSS files exist in the same directory or the correct path is specified. These imports are necessary for the variables and styles used in your SCSS files.